### PR TITLE
Add Protobuf plugin documentation

### LINKED ISSUE #51: Declare `.spine` directory in a single place
Now the `.spine` directory is declared all over the `model-compiler` module. It should be declared once and then reused.

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we add the `protoc-plugin/README.md` file with the documentation on the Spine Protobuf compiler plugin.

Also, this PR fixes #51.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: protoc-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Spine Protobuf compiler plugin
+
+As Spine relies heavily on Google Protobuf language, it is required  to introduce some kind of
```
Comment (armiol) [inline protoc-plugin/README.md]:
```
Spine uses Google Protobuf as a model definition language. Some of the framework features require compile-time processing of the data model. For that a plugin into a Protobuf compiler (a.k.a. `protoc`) is implemented.
```

File: protoc-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Similarly to `(is)` option, file-level `(every_is)` option declares that all the message types 
+declared in the current file implement the given interface.
+
+```proto
+import "spine/options.proto";
+
+option (every_is) = "org.example.Pet";
+
+message Cat {
+    // Remainder omitted.
```
Comment (armiol) [inline protoc-plugin/README.md]: `//...` looks shorter and have the same meaning.

File: protoc-plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+The Spine Gradle plugin automatically attaches the Spine `protoc` plugin to the Protobuf compilation
+process performed by Protobuf Gradle plugin.
+
+For that, the Spine `protoc` plugin artifact is fetched from a maven repository and copied into 
+the directory `.spine` found under the project root, so that the artifact relative path is 
+`<projectDir>/.spine/spine-protoc-plugin-X.X.X.jar`, where `X.X.X` is the version of the downloaded 
+artifact, which is equal to the version of the Spine Gradle plugin used.
+
+For the `protoc` integration the shell script [launchers](./plugin_runner.sh) are used.
```
Comment (armiol) [inline protoc-plugin/README.md]: Please also have `## How it works` technical section.

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Pull request merged**